lib/widgets/category_card.dart

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/models/category.dart';
+import 'package:flutter/material.dart';
+
+/// A card widget that displays a SampleCategory.
+class CategoryCard extends StatelessWidget {
+   const CategoryCard({
+    required this.category, required this.onClick, super.key,
```

Comment:
I think `dart format` needs to be run on this file. I think it would put these arguments on separate lines.